creator-node/src/middlewares.js

```diff
@@ -508,32 +508,34 @@ async function getOwnEndpoint({ libs }) {
 }
 
 /**
- * Retrieves user replica set from discprov
+ * Retrieves user replica set endpoints from discprov
  *
  * Polls discprov conditionally as follows:
  *    - If blockNumber provided, polls discprov until it has indexed that blocknumber (for up to 200 seconds)
  *    - Else if ensurePrimary required, polls discprov until it has indexed myCnodeEndpoint (for up to 60 seconds)
  *      - Errors if retrieved primary does not match myCnodeEndpoint
  *    - If neither of above conditions are met, falls back to single discprov query without polling
  *
+ * @param {string} wallet - wallet used to query discprov for user data
  * @param {Object} serviceRegistry
  * @param {Object} logger
- * @param {string} wallet - wallet used to query discprov for user data
  * @param {number} blockNumber - blocknumber of eth TX preceding CN call
  * @param {string} myCnodeEndpoint - endpoint of this CN
  * @param {boolean} ensurePrimary - determines if function should error if this CN is not primary
  *
  * @returns {Array} - array of strings of replica set
  */
-async function getCreatorNodeEndpoints({
+async function getUserReplicaSetEndpointsFromDiscovery({
```

Comment:
all changes here and below are just renaming getCreatorNodeEndpoints to getUserReplicaSetEndpointsFromDiscovery